sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/models/ImageLocation.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+package com.azure.ai.openai.implementation.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.Immutable;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** An image response item that provides a URL from which an image may be accessed. */
+@Immutable
+public final class ImageLocation {
```

Comment:
diff to models
https://github.com/Azure/azure-sdk-for-java/blob/4f129e536ceb079f89b0b682e80cc25c0df5dead/sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ImageLocation.java#L48-L51